README.md

```diff
@@ -1,26 +1,16 @@
-# README
+# Repository Mirroring Tool
 ![Kartoha](https://travis-ci.org/SUSE/rmt.svg?branch=master)
 
+This tool allows you to mirror RPM update repositories in your own private network. Registered organization credentials are required to mirror SUSE repositories.
```

Comment:
Better wording would be: credentials are required for SUSE Enterprise Linux repositories.
openSUSE repos are SUSE repos too.